README.md

```diff
@@ -4,6 +4,38 @@
 
 This repository holds various examples that showcases Screenly's features and API.
 
+## Using the Playground theme
+
+If your Edge App depends on the Playground theme, make sure to run the following command
+before uploading changes (via `screenly edge-app upload`):
+
+```bash
+EDGE_APP_PATH=<EDGE_APP_PATH> bash scripts/copy_playground_theme.sh
```

Comment:
Unnecessary complex. `./scripts/copy_playground_theme.sh` is fine with the requested changes to the script.

Reply:
If I were to run it inside the `Playground/` directory, it would never know where to copy it (e.g., Instagram, RSS feed reader).
But if I were to run it inside `edge-apps/rss-reader` (for example), I can modify the script so that it can be used like `../../scripts/copy_playground_theme.sh`. That way, the script can just make use of just `$(pwd)`, therefore not needing the `$EDGE_APP_PATH` anymore.